docs/resources/logdna_stream_config.md

```diff
@@ -0,0 +1,47 @@
+# Resource: `logdna_stream_config`
+
+> **IBM Log Analysis and Cloud Activity Tracker users only**
+
+Manages [LogDNA Streaming](https://ibm.github.io/cloud-enterprise-examples/log-streaming/content-overview/) configuration for an account.
+
+## Example
+
+```hcl
+provider "logdna" {
+  servicekey = "xxxxxxxxxxxxxxxxxxxxxxxx"
+  url = "https://api.logdna.com" # (Optional) specify a LogDNA region
+}
+
+resource "logdna_stream_config" "config" {
+  user      = var.stream_user
+  password  = var.stream_password
+  topic     = "example"
+  brokers   = [
+    "broker-1.example.org:9093",
+    "broker-2.example.org:9093",
+    "broker-3.example.org:9093",
+  ]
+}
+```
+
+## Import
+
+Importing an existing configuration is supported:
+
+```sh
+$ terraform import logdna_stream_config.config stream
```

Comment:
Based on your other comment, this has to be `stream`, which also has to match the const in the code, huh?

Reply:
yeah, so afaict there has to be some kind of unique identifier associated with a resource. it can be derived, or it can be returned from the API, but ultimately there needs to be something that uniquely identifies it in order to use it with `terraform import`. since there's nothing that can be used or derived from this configuration sub-object, and there's only ever 1 of these, I just picked `stream` as a static "ID"

Reply:
yep, seems like a good choice for a static id value